client/src/components/Form/Elements/FormNumber.vue

```diff
@@ -125,6 +150,15 @@ export default {
                 this.dismissCountDown = this.dismissSecs;
             }
         },
+        isNumberOrDecimal(event) {
+            // NOTE: Should we check for `fieldType` here?
+            const charCode = event.charCode;
+            if ((charCode >= 48 && charCode <= 57) || charCode === 46) {
```

Comment:
leave a comment here on what that does ? Also seems to be deprecated ? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/charCode

Is there maybe a better way to do this ?

Reply:
This is a better way of doing this:
```suggestion
            if ((key >= '0' && key <= '9') || key === '.') {
```

The reason this was done was to prevent non number (and decimal `.`) inputs.

Without this we have a tiny bug of being able to still enter characters like `-` or `+`:

https://github.com/user-attachments/assets/afe6ed63-a194-4275-afe0-c5eee70122cd

With this the input disallows absolutely everything except for numbers (and decimals: in case of a float input)

(Note: _flagging decimals as invalid for integer inputs works well and remains unchanged for this_)